Fix Cell.cs

atonce-original opened the pull request:
Fix: Added empty refs collection check in Cell constructor

Before computing the maximum level from the refs collection, a check is now performed to ensure that the collection is not empty. If the collection is empty, the level is set to 0 (explicitly cast to byte), preventing the "Sequence contains no elements" exception during cell construction.

justdmitry commented:
Good catch, thanks.
Please, replace `Any()` with `Count != 0` to conform https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1860

atonce-original commented:
Thanks for the feedback! I've made the requested changes:
- Added a check for an empty `refs` collection to prevent the "Sequence contains no elements" exception.
- Replaced `refs.Any()` with `refs.Count > 0` to comply with CA1860 recommendations and improve performance.

Please let me know if there are any further adjustments needed.

justdmitry commented:
Thanks! New version has been pushed to NuGet.
